specification/sections/05-events.md

```diff
@@ -137,3 +137,21 @@ See [provider initialization](./02-providers.md#24-initialization) and [setting
 > `PROVIDER_READY` handlers attached after the provider is already in a ready state **MUST** run immediately.
 
 See [provider initialization](./02-providers.md#24-initialization) and [setting a provider](./01-flag-evaluation.md#setting-a-provider).
+
+#### Requirement 5.3.4
+
+> When the provider's `on context changed` is called, `PROVIDER_STALE` handlers **MUST** run.
```

Comment:
Please update the `Provider Events` table to include the new event.

https://github.com/open-feature/spec/blob/main/specification/types.md#provider-events

Reply:
Whoops, you're right!